README.md

```diff
@@ -61,6 +61,12 @@ $ pip install -r requirements.txt
 $ python bootstrap.py login
 ```
 
+## Installation with Docker
+
+You can also use [Wand](https://github.com/solero/wand), a tool that installs Houdini, Dash and many other useful tools to compliment your experience.
```

Comment:
Perhaps add something that specifies that wand is only available to linux users at the moment?

Reply:
Who would run a CPPS on Windows anyway!

Reply:
Added